articles/native-platforms/ios-swift/_includes/_api_authz.md

```diff
@@ -0,0 +1,131 @@
+## Configuring Your Application
+
+## Dependences
+
+This quickstart uses the Auth0.swift SDK to help you add authentication and API authorization to your iOS app. You will need to add this dependency to your Podfile.
+
+```ruby
+use_frameworks!
+pod "Auth0", :git => 'https://github.com/auth0/Auth0.swift.git'
+```
+
+Then, run `pod install`.
+
+> For further reference on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
+
+## Configuration
+
+First, add a `Auth0.plist` file to your main bundle. This should contain the ClientId and Domain from your Auth0 tenant configuration:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+   <key>ClientId</key>
+   <string>{YOUR_CLIENT_ID}</string>
+   <key>Domain</key>
+   <string>{YOUR_DOMAIN}</string>
+</dict>
+</plist>
+```
+
+In your application's `Info.plist` file, register your iOS Bundle Identifer as a custom scheme:
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+    <dict>
+        <key>CFBundleTypeRole</key>
+        <string>None</string>
+        <key>CFBundleURLName</key>
+        <string>auth0</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+        </array>
+    </dict>
+</array>
+```
+
+Finally, open the Dashboard and make sure the Allowed Callback URLs for your client contains a URL with the following format:
+
+`{YOUR_BUNDLE_IDENTIFIER}://{YOUR_AUTH0_DOMAIN}/ios/{YOUR_BUNDLE_IDENTIFIER}/callback`
+
+## Initiate Authentication and Authorization
+
+Your Swift application will need to use Auth0.swift to kick off the OAuth flow using the code below:
+
+```swift
+@IBAction func StartAuthentication( sender: AnyObject? ) {
```

Comment:
camel case starting with lowercase for methods name, no spaces after `(` and before `)`